Update peer dependency version

Hello! Thank you for the nice icons.

I have switched to npm 7 and it installs peer dependencies by default. So when I updated Vue to `3.0.11` I got a conflict with `phosphor-vue`'s peer dependencies:

```
  "peerDependencies": {
    "vue": "^3.0.0 <3.0.3"
  },
  ```

Could you, please, change it to `^3.0.0` and make a new patch release?
Right now I can run `npm install` only with `--legacy-peer-deps` (when there is no `package-lock.json`).

I am facing the same issue and since upgrading to the latest npm version I need to install my dependencies using `npm install --force` or at least `npm install --legacy-peer-deps`: <https://stackoverflow.com/questions/66020820/npm-when-to-use-force-and-legacy-peer-deps>

Otherwise following error occurs:
```
npm ERR! Could not resolve dependency:
npm ERR! peer vue@"^3.0.0 <3.0.3" from phosphor-vue@4.4.0
npm ERR! node_modules/phosphor-vue
npm ERR!   dev phosphor-vue@"4.4.0" from the root project
```

Moreover, I am puzzled that the dependency of phosphor-vue to vue is listed in the devDependencies with `"vue": "^3.2.26"` and in peerDependencies with `"vue": "^3.0.0 <3.0.3"`. It does not match up!

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "vue": "^3.2.26"
   },
   "peerDependencies": {
-    "vue": "^3.0.0 <3.0.3"
+    "vue": "^3.0.0"
   },
   "engines": {
     "node": ">=10"

